documentation/PhotometryFilters.md

```diff
@@ -12,6 +12,7 @@ Columns marked with an asterisk (*) may not be empty.
 | width | width of band | Angstroms (10<sup>-10</sup>m) | |
 
 ## Notes
+- Useful functions: `ingest_photometry_filter`, `fetch_svo`, `assign_ucd`
```

Comment:
Not clear where these functions are defined, are these all `utils.simple`? Sounds like not for all of them.

Reply:
These are gonna move to `astrodb_scripts` so I'd like to leave vague for now.